core/common/src/main/java/org/phenotips/remote/common/internal/api/DefaultJSONToMatchingPatientConverter.java

```diff
@@ -43,22 +48,42 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DefaultJSONToMatchingPatientConverter implements JSONToMatchingPatientConverter
 {
-    private Logger logger;
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultJSONToMatchingPatientConverter.class);
 
     private final Pattern hpoTerm; // not static: may be different from api version to api version
 
     private final Pattern disorderTerm;
 
-    private final Vocabulary ontologyService;
+    private static final Vocabulary MIM_VOCABULARY;
```

Comment:
Does this pass checkstyle? AFAIR, all constants should come before other types of fields.